packages/sdk/src/evm/common/feature-detection.ts

```diff
@@ -23,6 +23,7 @@ import {
 import { ExtensionNotImplementedError } from "./error";
 import { ThirdwebStorage } from "@thirdweb-dev/storage";
 import { BaseContract, BigNumber, ethers } from "ethers";
+import multihashes from "multihashes";
```

Comment:
weird that this changed anything

Reply:
agree, i'd love someone else to verify this was needed.